pkg/compose/convergence.go

```diff
@@ -513,6 +513,16 @@ func (s *composeService) createMobyContainer(ctx context.Context,
 		}
 		plat = &p
 	}
+
+	links, err := s.getLinks(ctx, project.Name, service, number)
+	if err != nil {
+		return created, err
+	}
+	for k, s := range networkingConfig.EndpointsConfig {
+		s.Links = links
```

Comment:
using NetworkConnect with network name this wasn't required. Sounds like some engine weirdness, anyway this is consistent we define NetworkConfig the same way we later connect to network